staging/src/k8s.io/component-base/config/options/leaderelectionconfig.go

```diff
@@ -42,8 +42,7 @@ func BindLeaderElectionFlags(l *config.LeaderElectionConfiguration, fs *pflag.Fl
 		"of a leadership. This is only applicable if leader election is enabled.")
 	fs.StringVar(&l.ResourceLock, "leader-elect-resource-lock", l.ResourceLock, ""+
 		"The type of resource object that is used for locking during "+
-		"leader election. Supported options are 'leases', 'endpointsleases' "+
-		"and 'configmapsleases'.")
+		"leader election. Supported options is 'leases'.")
```

Comment:
Should this be changed? We're not *using* the legacy resource locks, but they are still available for any controllers configured to use them and so should remain documented?

Reply:
They are not available and will throw an error if a non lease resource lock is used: https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/client-go/tools/leaderelection/resourcelock/interface.go#L112-L125